lib/testPaymentExpiry.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+const _ = require('lodash')
+const moment = require('moment')
+const request = require('co-request')
+const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
+
+module.exports = function * (config, payment) {
+  let tester = new TransferTester(config, payment)
+  yield tester.loadCaseExpiries()
+  return tester
+}
+
+function TransferTester (config, payment) {
+  this.config = config
+  this.source_transfers = payment.source_transfers
```

Comment:
Ok, it isn't completely arbitrary. I use snake case where it it is going to a JSON object, or where it is from one (or from the schema). This way, you can search e.g. `/source_transfers`, not `/source_transfers\|sourceTransfers`.